installation/6.docker.md

```diff
@@ -0,0 +1,201 @@
+In this section we explain how to setup MAD and a Rocketmap database using docker-compose.
+
+### 1.1 Preparations:
+
+You can just copy & paste this to do what is written below: 
+```
+mkdir MAD-docker && \
+cd MAD-docker && \
+mkdir mad && \
+mkdir mad/configs && \
+touch docker-compose.yml && \
+wget https://gist.githubusercontent.com/sn0opy/fb654915180cfbd07d5a30407c286995/raw/8467212e1371cc3f6a385e0bc7c3f63daa7a488a/rocketmap-osm-cec.sql && \
+cd mad/configs/ && \
+wget -O config.ini https://raw.githubusercontent.com/Map-A-Droid/MAD/master/configs/config.ini.example && \
+wget -O mappings.json https://raw.githubusercontent.com/Map-A-Droid/MAD/master/configs/mappings_example.json  && \
+mkdir geofences && cd ../../
+```
+This will: 
+1. Create a directory `MAD-docker`. 
+2. Create a file `docker-compose.yml`.
+3. Create a directory `MAD-docker/mad`. (here we store MAD related stuff)
+4. Create a directory `MAD-docker/mad/configs`. (here we store config files for MAD).
+Here you store your `config.ini`, `mappings.json` and a directory `geofences` (which contains your geofences).
+Examples for these files can be found @github https://github.com/Map-A-Droid/MAD/tree/master/configs
+5. Download the Rocketmap Database Schema: https://gist.github.com/sn0opy/fb654915180cfbd07d5a30407c286995 and store it
+as a file `rocketmap-osm-cec.sql`
+
+Your directory should now look like this:
+ ``` 
+ --MAD-docker/
+  |-- docker-compose.yml
+  |-- rocketmap-osm-cec.sql
+  |-- mad/
+    |-- configs/
+      |-- config.ini
+      |-- mappings_.json
+      |-- geofences/
+ ```
+ 
+### 1.2 Writing the docker-compose file.
+We use `docker-compose` to deploy and manage our services.
+
+Fill `docker-compose.yml` with the following content. Below we explain the details of every service.
+```yaml
+version: '2.4'
+services:
+  mad:
+    container_name: pokemon_mad
+    image: mapadroid/map-a-droid
+    restart: always
+    volumes:
+      - /etc/timezone:/etc/timezone:ro
+      - /etc/localtime:/etc/localtime:ro
+      - ./mad/configs/geofences:/usr/src/app/configs/geofences
+      - ./mad/configs/config.ini:/usr/src/app/configs/config.ini
+      - ./mad/configs/mappings.json:/usr/src/app/configs/mappings.json
+      - ./volumes/mad/version.json:/usr/src/app/version.json
+      - ./volumes/mad/files:/usr/src/app/files
+      - ./volumes/mad/logs:/usr/src/app/logs
+    depends_on:
+      - rocketdb
+    networks:
+      - default
+    ports:
+      - "5555:8080"
+      - "5554:8000"
+      - "5000:5000"
+
+  rocket-db:
+    container_name: pokemon_rocketdb
+    image: mariadb:10.3
+    restart: always
+    command: ['mysqld', '--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+    environment:
+      MYSQL_ROOT_PASSWORD: StrongPassword
+      MYSQL_DATABASE: rocketdb
+      MYSQL_USER: rocketdb
+      MYSQL_PASSWORD: AnotherStrongPassword
+      TZ: Europe/Berlin
+    volumes:
+      - ./volumes/rocketdb:/var/lib/mysql
```

Comment:
You can add a second volume containing the SQL file:
`- ./DIRECTORYWITHSQLFILEINIT/:/docker-entrypoint-initdb.d/`
That will execute every sql file in it, ordered by alphabet. But only one time of course :D 

See [here](https://github.com/docker-library/docs/tree/master/mariadb#initializing-a-fresh-instance)

Reply:
fixed with 897ca73